Remove references to pager

Index: dictd/colorit.1.in
===================================================================
--- dictd.orig/colorit.1.in	2008-01-23 23:39:37.000000000 +0100
+++ dictd/colorit.1.in	2008-01-23 23:41:02.000000000 +0100
@@ -11,9 +11,8 @@
 .fi
 .SH "DESCRIPTION"
 .B colorit
-is a script for markuping text input and sending a result to pager (
-.I less \-r
-by default) or stdout.
+is a script for markuping text input and sending a result to
+stdout.
 Markuping rules are described in a configuration file
 which is filtered by preprocessor (the default is
 .I m4
@@ -25,9 +24,7 @@
 .I cpp.
 This script can be used for colorizing the text by
 ANSI escape sequences, or making a simple text conversions
-and may be used as a pager instead of
-.I less
-by
+and may be used for colorizing output of 
 .I dict
 or other programs. In particular, log files,
 .I dict,
@@ -50,10 +47,6 @@
 or
 .I @SYSCONFDIR@/colorit.conf
 .TP
-.BI \-P " pager\fR or "  \-\-pager " pager"
-Specify a pager (- for none). The default is
-.I less -r
-.TP
 .BI \-\-pp " preprocessor"
 Specify a preprocessor (- for none).
 .SH "FILES"
