Patch for  #378792: dictl should use iconv's //translit suffix


Index: dictd/dictl.in
===================================================================
--- dictd.orig/dictl.in	2009-03-08 16:54:58.000000000 +0100
+++ dictd/dictl.in	2009-03-08 16:55:53.000000000 +0100
@@ -20,7 +20,7 @@
 fi
 
 internal_iconv (){
-    iconv -c -f $1 -t $2
+    iconv -c -f $1 -t $2//TRANSLIT
 }
 
 internal_recode (){
