Fix spelling typos found by lintian.

Index: dictd/daemon.c
===================================================================
--- dictd.orig/daemon.c	2011-01-23 10:07:57.000000000 +0100
+++ dictd/daemon.c	2011-01-23 10:08:22.000000000 +0100
@@ -253,7 +253,7 @@
       return DICT_DENY;
    }
    if (!*pt) {
-      log_info( ":E: Misformed range %s, denying access to %s\n", spec, ip);
+      log_info( ":E: Malformed range %s, denying access to %s\n", spec, ip);
       return DICT_DENY;
    }
    
Index: dictd/dictd.8.in
===================================================================
--- dictd.orig/dictd.8.in	2011-01-23 10:08:09.000000000 +0100
+++ dictd/dictd.8.in	2011-01-23 10:08:22.000000000 +0100
@@ -1009,7 +1009,7 @@
 .I lev
 , the strategy
 .I word
-may be prefered for databases mainly containing
+may be preffered for databases mainly containing
 the multiword phrases but the single words.
 .RE
 .TP
@@ -1330,13 +1330,13 @@
 especially
 .P
 .BI 00\-database\-info
-Containts the informarion about database
+Contains the informarion about database
 which is 
 returned by SHOW INFO command,
 unless it is specified in the configuration file.
 .P
 .BI 00\-database\-short
-Containts the short name of the database
+Contains the short name of the database
 which is
 returned by SHOW DB command,
 unless it is specified in the configuration file.
Index: dictd/dictfmt.1.in
===================================================================
--- dictd.orig/dictfmt.1.in	2011-01-23 10:08:15.000000000 +0100
+++ dictd/dictfmt.1.in	2011-01-23 10:08:22.000000000 +0100
@@ -278,7 +278,7 @@
 the first column can be treated as an index column
 (where the MATCH command searches) and the fourth column as a result column
 (where the MATCH gets things to be returned),
-and they (1-st and 4-th columns) are completely independant of each other.
+and they (1-st and 4-th columns) are completely independent of each other.
 The default value for this separator is ASCII symbol " \\034".
 .TP
 .BI \-\-break\-headwords
Index: dictd/dictfmt.c
===================================================================
--- dictd.orig/dictfmt.c	2011-01-23 10:07:57.000000000 +0100
+++ dictd/dictfmt.c	2011-01-23 10:08:22.000000000 +0100
@@ -840,7 +840,7 @@
                      Example: autumn%%%fall can be used\n\
                      if '--headword-separator %%%' is supplied",
 "--index-data-separator <sep> sets index/data separator which allows\n\
-                     to explicitely set fourth column in .index file,\n\
+                     to explicitly set fourth column in .index file,\n\
                      the default is \"\\034\"",
 "--break-headwords    multiple headwords will be written on separate lines\n\
                      in the .dict file.  For use with '--headword-separator.",
